src/meltano/core/tracking/__init__.py

```diff
@@ -1 +1,172 @@
-from .ga_tracker import GoogleAnalyticsTracker
+"""Meltano telemetry."""
```

Comment:
I've been trained over time to keep `__init__.py` files as sparse as possible, reserving them mostly just for imports.

Wdyt of moving the bulk of this to something like `/src/meltano/core/tracking/tracker.py`?

Reply:
Will do